redhat-developer/developers.redhat.com
# DEVELOPER 5332 - Nav search redesign

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-5332

### Verification Process

1. Go to any page on desktop
2. Click the magnifying glass in the nav
3. Verify the search bar pops up
4. Click the 'x' on the search bar
5. Verify that closes the search bar
6. Open the search bar again
7. Click anywhere else on the page
8. Verify clicking elsewhere closes the search bar
9. Open the search bar
10. Enter a search term
11. Hit enter
12. Verify you are taken to a relevant search page

### Demo

![sep-18-2018 11-23-54](https://user-images.githubusercontent.com/10933938/45698275-a161c480-bb35-11e8-90af-46e116f283f5.gif)
